ee/tabby-webserver/src/repositories/mod.rs

```diff
@@ -12,20 +12,19 @@ use axum::{
 };
 use hyper::Body;
 use juniper_axum::extract::AuthBearer;
+pub use resolve::RepositoryCache;
 use tabby_common::config::RepositoryConfig;
 use tracing::{instrument, warn};
 
 use crate::{
-    repositories::resolve::{
-        contains_meta, resolve_all, resolve_dir, resolve_file, resolve_meta, RepositoryMeta,
-        ResolveParams,
-    },
+    repositories::resolve::{RepositoryMeta, ResolveParams},
     schema::auth::AuthenticationService,
 };
 
 #[derive(Debug)]
 pub struct ResolveState {
     pub repositories: Vec<RepositoryConfig>,
```

Comment:
would you think it make sense to simply refactor the `repositories` field's functionality into part of `RepositoryCache`? It'll be easier adding functionalities like loading from database in future (single struct `RepositoryCache` manage all the inputs), and the code structure for this routes seems better to me.

Reply:
Yeah, I do like the sound of that.